Content.Server/Voting/Managers/VoteManager.DefaultVotes.cs

```diff
@@ -61,7 +62,7 @@ private void CreateRestartVote(ICommonSession? initiator)
             foreach (var player in _playerManager.Sessions)
             {
                 _playerManager.UpdateState(player);
-                if (player.Status != SessionStatus.Disconnected && _entityManager.HasComponent<GhostComponent>(player.AttachedEntity))
+                if (player.Status != SessionStatus.Disconnected && ( _entityManager.HasComponent<GhostComponent>(player.AttachedEntity) || _entityManager.HasComponent<ZombieComponent>(player.AttachedEntity)))
                 {
                     ghostCount++;
```

Comment:
Coupling this tightly is terrible for forks who do not use zombies, use the eventbus as it's why we have it.